schemas/backend_configuration_schema.json

```diff
@@ -202,6 +202,28 @@
                     "minItems": 1,
                     "description": "Frequency range for the measurement LO",
                     "items": {"type": "array", "minItems": 2, "maxItems": 2, "items": {"type": "number"}}
+                },
+                "timing_constraints": {
+                    "type": "object",
+                    "required": ["granularity", "min_length", "pulse_alignment", "acquire_alignment"],
```

Comment:
Are these all required by the passes or can they be supplied individually?

Reply:
I assumed all IBM devices will provide them, however, in transpiler configuration they have a default value of 1. So not necessary be provided. Empty dict can work. https://github.com/Qiskit/ibm-quantum-schemas/pull/11/commits/84dddfba1f67e3c25bfc8b6ab29f6124ceeaf3d3